ui/libs/react-renterd/src/useRhp.ts

```diff
@@ -0,0 +1,7 @@
+// mux.POST("/api/rhp/scan", srv.rhpScanHandler)
+// mux.POST("/api/rhp/form", srv.rhpFormHandler)
+// mux.POST("/api/rhp/renew", srv.rhpRenewHandler)
+// mux.POST("/api/rhp/read", srv.rhpReadHandler)
+// mux.POST("/api/rhp/append", srv.rhpAppendHandler)
+
+export const foo = 1
```

Comment:
what is this file?

Reply:
Its for rhp APIs which aren't implemented yet, I'll add them.